### ISSUE TITLE
`ValidationError` when showing associated interactions

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#3464]
https://vimsfish-v789.test.specifysystems.org/specify/view/collectionobject/23039/

Clicking on the `SHOW_LOANS` button leads to an error:

https://user-images.githubusercontent.com/37256050/236903882-b6e93edd-92d0-4c83-a12b-86da29e4f854.mov

```
ValidationError at /api/specify/loanpreparation/ ['“true” value must be either True or False.']
```

```
ValidationError at /api/specify/loanpreparation/ ['“false” value must be either True or False.'] Request Method: GET Request URL: http://vimsfish-v789.test.specifysystems.org/api/specify/loanpreparation/?isresolved=false&limit=20&preparation=23606 Django Version: 3.2.15 Python Executable: /opt/specify7/ve/bin/python3.8 Python Version: 3.8.0 Python Path: ['/opt/specify7', '/opt/specify7', '/opt/specify7/ve/bin', '/usr/lib/python38.zip', '/usr/lib/python3.8', '/usr/lib/python3.8/lib-dynload', '/opt/specify7/ve/lib/python3.8/site-packages', '/opt/specify7'] Server time: Mon, 08 May 2023 18:35:51 -0500 Installed Applications: ('django.contrib.sessions', 'django.contrib.staticfiles', 'django.contrib.contenttypes', 'django.contrib.auth', 'specifyweb.specify', 'specifyweb.permissions', 
```

[Specify 7 Crash Report - 2023-05-08T18_36_13.836Z.txt](https://github.com/specify/specify7/files/11424098/Specify.7.Crash.Report.-.2023-05-08T18_36_13.836Z.txt)

**Comment by grantfitzsimmons** [comment]
This is fixed in `v7.8.10`, but not working in `v7.8.9`. Why is that?

**Comment by maxxxxxdlp** [comment]
it was fixed in https://github.com/specify/specify7/pull/3298
